todo/services/label_service.py

```
@@ -0,0 +1,91 @@
+                    page=page,
+                    limit=limit,
+                    links=None,
+                )
+
+            label_dtos = [cls.prepare_label_dto(label) for label in labels]
+
+            links = cls.prepare_pagination_links(page=page, total_pages=total_pages, limit=limit, search=search)
+
+            return GetLabelsResponse(labels=label_dtos, total=total_count, page=page, limit=limit, links=links)
+
+        except Exception:
```

Pylint (3.3.7)

[warning] Catching too general exception Exception

(W0718)